tests/integration/_experimental/dataset/test_evaluation.py

```diff
@@ -148,6 +173,48 @@ def test__test__multiple_eval_configs() -> None:
     assert fetched_eval_config_2 == eval_config_2
     expected_df_result_1 = df_result_1.drop(columns=[JOIN_COLUMN])[3:10].reset_index(drop=True)
     expected_df_result_2 = df_result_2.drop(columns=[JOIN_COLUMN])[3:10].reset_index(drop=True)
+    result_columns_1.remove(JOIN_COLUMN)
```

Comment:
I think we should restore id columns also for result. We dropped it behind users' back, when we return it to the users, it should look like the inputs they gave us.

cc @yifanwu-kolena

Reply:
This is existing behavior, so punting to future discussion if there's user feedback.